lib/mix/tasks/generate_resource_diagrams.ex

```diff
@@ -52,7 +52,7 @@ defmodule Mix.Tasks.Ash.GenerateResourceDiagrams do
                 "mermaid-er-diagram",
                 format,
                 Ash.Api.Info.Diagram.mermaid_er_diagram(api),
-                "Generated ER Diagram for #{inspect(api)}"
+                "Generated ER Diagram for #{inspect(api)} (as #{format})"
```

Comment:
Maybe showing the file path & name here is useful as well? Potentially instead of the format, since the file extension will tell them?

Reply:
good point